### ISSUE TITLE
errorf improvements

### ISSUE THREAD

**Issue by ccoVeille** [catenacyber/perfsprint#36]
I'm still surprised by the ability of people to create arrange code

fmt.Errorf("%s", "whatever")

https://github.com/search?q=language%3Ago++%22fmt.Errorf%28%5C%22%25s%5C%22%2C+%5C%22%22&type=code

fmt.Errorf("%w", err)

https://github.com/search?q=language%3Ago++%22fmt.Errorf%28%5C%22%25w%5C%22%2C+%22&type=code

These are somehow people needing sprint1 with fmt.Errorf

_Originally posted by @ccoVeille in https://github.com/mgechev/revive/pull/1136#discussion_r1850947674_

**Comment by catenacyber** [comment]
Indeed, `fmt.Errorf("%s", "whatever")` is not tested and seems not handled.
Will you work on this ?

**Comment by catenacyber** [comment]
But not every linter agrees it seems : https://github.com/go-critic/go-critic/issues/1381

**Comment by ccoVeille** [comment]
The fact gocritic doesn't support it, allow you to add it 😅

**Comment by ccoVeille** [comment]
We could also add

fmt.Errorf("%s", err)
https://github.com/search?q=language%3Ago++%22fmt.Errorf%28%5C%22%25s%5C%22%2C+err%22&type=code

But this one might be already reported

**Comment by ccoVeille** [comment]
> Will you work on this ?

Not yet, it was more a message in the bottle, to add it later.

Anyone can work on it
